src/components/shared/Swiper.tsx

```diff
@@ -0,0 +1,329 @@
+/**
```

Comment:
General note on this, it does seem like this is a good deal more general and flexible than what we need in our own codebase (the number of properties you can pass it is quite expansive and technical!) but given that it works let's just keep it; saying it with an eye to keep the complexity of our codebase down. Perhaps we can just expose a subset of this configurability through a higher level, simpler component.